app/src/androidTest/java/de/test/antennapod/ui/DialogsTest.java

```diff
@@ -0,0 +1,98 @@
+package de.test.antennapod.ui;
+
+import android.content.Intent;
+import android.view.View;
+import androidx.test.espresso.Espresso;
+import androidx.test.espresso.contrib.RecyclerViewActions;
+import androidx.test.espresso.intent.rule.IntentsTestRule;
+import androidx.test.runner.AndroidJUnit4;
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.activity.MainActivity;
+import de.danoeh.antennapod.fragment.AddFeedFragment;
+import de.danoeh.antennapod.fragment.DownloadsFragment;
+import de.danoeh.antennapod.fragment.QueueFragment;
+import de.danoeh.antennapod.fragment.SubscriptionFragment;
+import de.test.antennapod.EspressoTestUtils;
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.clearText;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static androidx.test.espresso.action.ViewActions.scrollTo;
+import static androidx.test.espresso.action.ViewActions.typeText;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.RootMatchers.isDialog;
+import static androidx.test.espresso.matcher.ViewMatchers.isDescendantOfA;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.isRoot;
+import static androidx.test.espresso.matcher.ViewMatchers.withClassName;
+import static androidx.test.espresso.matcher.ViewMatchers.withContentDescription;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static de.test.antennapod.EspressoTestUtils.waitForView;
+import static de.test.antennapod.NthMatcher.first;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.endsWith;
+
+/**
+ * User interface tests for queue fragment.
+ */
+@RunWith(AndroidJUnit4.class)
+public class DialogsTest {
+
+    @Rule
+    public IntentsTestRule<MainActivity> activityRule = new IntentsTestRule<>(MainActivity.class, false, false);
+
+    @Before
+    public void setUp() throws InterruptedException {
+        EspressoTestUtils.clearPreferences();
+        EspressoTestUtils.clearDatabase();
+        EspressoTestUtils.setLastNavFragment(AddFeedFragment.TAG);
+        activityRule.launchActivity(new Intent());
+
+        String url = "https://omny.fm/shows/silence-is-not-an-option/why-not-being-racist-is-not-enough";
```

Comment:
We already have a class `UITestUtils` that automatically adds and hosts a podcast. I would prefer that over a real podcast because it makes us less dependent on somebody else. See `PlaybackTest` for example usage.

Reply:
I'm gonna have a look in that to update the tests :)